src/dotnet/Notification.Service/Db/DbDevice.cs

```diff
@@ -0,0 +1,40 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using ActualChat.Notification.Backend;
+using Microsoft.EntityFrameworkCore;
+using Stl.Versioning;
+
+namespace ActualChat.Notification.Db;
+
+[Table("Devices")]
+[Index(nameof(UserId))]
+public class DbDevice : IHasId<string>, IHasVersion<long>
+{
+    private DateTime _createdAt;
+    private DateTime _accessedAt;
+
+    public DbDevice() { }
+
+    [Key] public string Id { get; set; } = null!;
+
+    [ConcurrencyCheck]
+    public long Version { get; set; }
+
+    public string UserId { get; set; } = null!;
```

Comment:
So this is the field we'll be sharding by. Am I right `Id` includes `UserId` here?